--- origsrc/libgdata-0.16.1/gdata/gdata.symbols	2014-11-09 16:43:07.432734000 -0600
+++ src/libgdata-0.16.1/gdata/gdata.symbols	2015-01-04 22:18:07.076574100 -0600
@@ -1089,8 +1089,6 @@ gdata_freebase_topic_value_is_image
 gdata_freebase_topic_result_get_type
 gdata_freebase_topic_result_new
 gdata_freebase_topic_result_dup_object
-gdata_freebase_result_error_get_type
-gdata_freebase_result_error_quark
 gdata_freebase_result_get_type
 gdata_freebase_result_new
 gdata_freebase_result_dup_variant
